notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
```

Comment:
<p>You don't need to poll. Just call <code>ann_index.result()</code>.</p><p>Furthermore, you can use the Python SDK for Vertex AI instead of the GAPIC API as you did here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>The current Python SDK's create index does not support set index update method...</p>